.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
```

Comment:
These `.idea` files snuck in again, why doesn't the root `.gitignore` keep these out?

Reply:
IDK. Annoying.